pkg/cli/install.go

```diff
@@ -20,6 +20,8 @@ acorn install`,
 }
 
 type Install struct {
+	DisablePreflightChecks bool `usage:"Disable preflight checks"`
```

Comment:
Can we just change this to "checks' not have disable in there.  So `--checks=false` to disable it.  You need to change the field type to `*bool` which should work and then treat nil as true.

Reply:
Done :heavy_check_mark: